frontend/src/deviceDetailsPage/__tests__/DeviceDetailsTabPanels.test.jsx

```diff
@@ -14,10 +15,13 @@ jest.spyOn(global.console, "error");
 describe("DeviceDetailsPanels", () => {
   let wrapper;
   describe("ActivityPanel", () => {
-    it("Renders one Container component containing TODO comment", () => {
-      wrapper = Enzyme.shallow(<DeviceDetailsActivityPanel />);
+    const sampleDevice = SampleData.getSampleSender();
```

Comment:
Do you need to call `getSampleSender()`? Can you just create a dummy device info here instead?

Reply:
I saw that there was already a getSampleSender function in SampleData so I decided to utilize it. However, if it's better convention to just define a dummy device in the test itself, then I can do that instead.

Reply:
Local dummy data is better